lib/chef/resource/archive_file.rb

```diff
@@ -81,6 +81,10 @@ class ArchiveFile < Chef::Resource
         description: "Should the resource overwrite the destination file contents if they already exist? If set to `:auto` the date stamp of files within the archive will be compared to those on disk and disk contents will be overwritten if they differ. This may cause unintended consequences if disk date stamps are changed between runs, which will result in the files being overwritten during each client run. Make sure to properly test any change to this property.",
         default: false
 
+      property :strip_components, Integer,
```

Comment:
If you think we can get this done in the next few days add this:

`introduced: '17.5'` 

That powers our docs site.